### PR TITLE
Build: Bump Parquet to 1.15.0

### PR DESCRIPTION
**Pull request by Fokko**
For the full changelog, please check the pre-release:

https://github.com/apache/parquet-java/releases/tag/apache-parquet-1.15.0-rc1

### CODE REVIEW

**Comment by Fokko** [comment]
Seems that JDK22 is not yet supported :(

**Comment by Fokko** [comment]
It already landed in Spark: https://github.com/apache/spark/pull/48970

File: build.gradle [hunk 1]
```diff
@@ -119,6 +119,9 @@ allprojects {
   repositories {
     mavenCentral()
     mavenLocal()
+    maven {
+      url = uri("https://repository.apache.org/content/repositories/orgapacheparquet-1067")
+    }
```
Comment (nastra) [inline build.gradle:124]:
```suggestion
```

**Review by nastra — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (Fokko) [reply build.gradle]: Oops, I thought I already removed that 🤦 Thanks for cleaning this up

**Pull request merged**